Logger.lua

```diff
@@ -92,11 +83,15 @@ function Logger:new(name, dir, console)
   if not dir or  #dir == 0 then o.Path = "./"
   elseif dir and DirExists(dir) then o.Path = DirNormalize(dir)
   elseif dir and not DirExists(dir) then
-    error("Path '" .. dir .. "' doesn't exists or you can't have permissions to use it.")
+    error("Path '" .. dir .. "' doesn't exists or you don't have permissions to use it.")
   else -- idk...
-    error("Something's wrong with '"..dir.."'... (argument #2 in 'new()')")
+    error("Unknown error while checking (and/or loading) '"..dir.."'... (argument #2 in 'new()')")
   end
 
+  local file = io.open(o.Path .. Fmt.File.Name:format(o.Namespace, os.date(Fmt.File.Suffix)), "a+")
```

Comment:
Added this thing that adds a header to the log everytime new() is called

añadi esta cosa que añade un header a el log cada vez que se corre new()